internal/kubeconfig/certificate.go

```diff
@@ -148,9 +148,13 @@ func createCSR(username string, privateKey *rsa.PrivateKey) []byte {
 func getCaBase64() string {
 	config, err := runtime.GetConfig()
 	if err != nil {
-		log.Printf("Unable to get kubeconfig.\n%v", err)
-		os.Exit(1)
+		log.Fatalf("Unable to get kubeconfig.\n%v", err)
 	}
 
-	return base64.StdEncoding.EncodeToString(config.CAData)
+	CAData, err := ioutil.ReadFile(config.CAFile)
+	if err != nil {
+		log.Fatalf("Unable to read kubeconfig file.\n%v", err)
```

Comment:
We have to test running it outside a kind cluster (locally with go run...) and a kubeconfig file instead of a serviceaccount. 

commented just to keep it in mind